org.eclipse.jdt.ui/core refactoring/org/eclipse/jdt/internal/corext/refactoring/rename/RenameFieldProcessor.java

```diff
@@ -857,7 +857,7 @@ private RefactoringStatus createChanges(IProgressMonitor pm) throws CoreExceptio
 		if (fIsRecordComponent) {
 			addAccessorOccurrences(new SubProgressMonitor(pm, 1), result);
 			if (fRenameLocalVariableProcessor != null) {
-				addLocalVariableOccurrences(new SubProgressMonitor(pm, 1), getNewElementName(), result);
```

Comment:
Eclipse auto-removed this parameter and changed the addLocalVariableOccurrences method signature because the progress monitor was unused in that function.